src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamAllowRenegotiationTests.cs

```diff
@@ -70,6 +70,7 @@ public async Task SslStream_AllowRenegotiation_True_Succeeds()
 
         [Fact]
         [OuterLoop] // Test hits external azure server.
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/68206", TestPlatforms.Android)]
```

Comment:
May be worth or re-testing. This test was failing on all platform recently because of server changes. It should be fixed now.  https://github.com/dotnet/runtime/issues/77414

Reply:
I'll re-test it, thanks for pointing that out!

Reply:
The test is passing.